patches/chrome-android-expectations-lint-suppressions.xml.patch

```diff
@@ -1,10 +1,10 @@
 diff --git a/chrome/android/expectations/lint-suppressions.xml b/chrome/android/expectations/lint-suppressions.xml
-index 75e5870bd2a12cf4adfce3e103cc8d3508c492d1..cff1d9b79a07e79d767ee4e3a8d5f9e8e6fbe796 100644
+index 75e5870bd2a12cf4adfce3e103cc8d3508c492d1..7e83dab8311ce9a76f80f2a5b16a069051e1b164 100644
 --- a/chrome/android/expectations/lint-suppressions.xml
 +++ b/chrome/android/expectations/lint-suppressions.xml
 @@ -268,4 +268,5 @@ https://chromium.googlesource.com/chromium/src/+/main/build/android/docs/lint.md
      <!-- Discussed in crbug.com/1069204, ignoring this class of errors since these are Q+ constants. -->
      <ignore regexp="Must be one of: LineBreaker.BREAK_STRATEGY_SIMPLE, LineBreaker.BREAK_STRATEGY_HIGH_QUALITY, LineBreaker.BREAK_STRATEGY_BALANCED"/>
    </issue>
-+  <issue id="DuplicateDefinition"><ignore regexp="chrome/android/java/res/values/brave_colors.xml"/></issue><issue id="StringFormatCount"><ignore regexp="sponsored_images_bottom_sheet_text"/><ignore regexp="notification_marketing"/><ignore regexp="renew_monthly_save"/><ignore regexp="wallet_swap_fees_notice"/><ignore regexp="processing_fee"/></issue><issue id="WrongConstant"><ignore regexp="ContentSettingsType.BRAVE_CONTENT_SETTINGS_TYPES_LIST"/></issue><issue id="UnusedResources"><ignore regexp="The resource `R.string.playlist_.*` appears to be unused"/></issue>
++  <issue id="DuplicateDefinition"><ignore regexp="chrome/android/java/res/values/brave_colors.xml"/></issue><issue id="StringFormatCount"><ignore regexp="sponsored_images_bottom_sheet_text"/><ignore regexp="notification_marketing"/><ignore regexp="renew_monthly_save"/><ignore regexp="wallet_swap_fees_notice"/><ignore regexp="processing_fee"/></issue><issue id="WrongConstant"><ignore regexp="ContentSettingsType.BRAVE_CONTENT_SETTINGS_TYPES_LIST"/></issue><issue id="UnusedResources"><ignore regexp="The resource `R.string.playlist_.*` appears to be unused"/></issue><issue id="UnsafeOptInUsageError"><ignore regexp='(markerClass = androidx.media3.common.util.UnstableApi.class)'/></issue>
```

Comment:
Added a patch to ignore `UnstableApi` based on https://developer.android.com/guide/topics/media/media3/getting-started/migration-guide#improvements_in_media3